content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -4,6 +4,7 @@ reviewers:
 title: Process ID Limits And Reservations
 content_type: concept
 weight: 40
+#toc_hide: true
```

Comment:
Why comment it out? Consider setting this to `false` instead (or, omit the line).